browser_tests/interaction.spec.ts

```diff
@@ -130,6 +131,28 @@ test.describe('Node Interaction', () => {
       }
     })
     await expect(comfyPage.canvas).toHaveScreenshot('prompt-dialog-closed.png')
+
+    const textWidgetPos = {
```

Comment:
Can this be a separate test case?

Reply:
Sorry, was really behind on notifications. Thanks for doing that.